pkg/github/github.go

```diff
@@ -5,10 +5,34 @@ import (
 	"fmt"
 	"log"
 	"strings"
+	"time"
 
+	"github.com/cyberark/conjur-oss-suite-release/pkg/changelog"
 	"github.com/cyberark/conjur-oss-suite-release/pkg/http"
+	"github.com/cyberark/conjur-oss-suite-release/pkg/repositories"
+	"github.com/cyberark/conjur-oss-suite-release/pkg/version"
 )
 
+// SuiteComponent represents a suite component with all of its changelogs and
+// relevant pin data
+type SuiteComponent struct {
+	CertificationLevel string
+	Changelogs         []*changelog.VersionChangelog
+	ReleaseName        string
+	ReleaseDate        string
+	Repo               string
+	UpgradeURL         string
+	URL                string
+}
+
+// ReleaseSuite stores all the data needed for generation of templates in the suite
+type ReleaseSuite struct {
+	Version          string
+	Date             time.Time
+	Components       []SuiteComponent
+	UnifiedChangelog string
+}
```

Comment:
This is used for templates and it doesn't go 1:1 w/ github apis so it should really be in template pkg

Reply:
Agreed - moved back to templates